docs/design.md

```diff
@@ -107,7 +107,6 @@ struct image_tlv {
 #define IMAGE_TLV_KEYHASH           0x01   /* hash of the public key */
 #define IMAGE_TLV_SHA256            0x10   /* SHA256 of image hdr and body */
 #define IMAGE_TLV_RSA2048_PSS       0x20   /* RSA2048 of hash output */
-#define IMAGE_TLV_ECDSA224          0x21   /* ECDSA of hash output */
```

Comment:
Don't remove, just comment it's not supported anymore.

Reply:
I've put it back, and marked it with not supported anymore.